CarLot/Program.cs

```diff
@@ -0,0 +1,133 @@
+            Truck truck2 = new Truck ("Raptor", "DJ955TF", "Chevrolet", "Colorado", 9, 650000);
+
+            Scottelder.addVehicle(car1);
+            Covert.addVehicle(truck1);
+            Covert.addVehicle(car2);
+            Scottelder.addVehicle(truck2);
+            Covert.lotInventory();
+            Scottelder.lotInventory();
+
+            Console.WriteLine ("{0} has these vehicles.", Scottelder.name);
+            Scottelder.lotInventory();
+            Console.WriteLine ("");
+
+            Console.WriteLine ("{0} has these vehicles.", Covert.name);
+            Covert.lotInventory();
+            Console.WriteLine ("");
+
+        }
+    }
+
+    //Listing Names and Vehicles
+    class CarLot {
+        public string name = "";
+        List<Vehicle> numofCars = new List<Vehicle> ();
+
+        public CarLot (string myName) {
+            this.name = myName;
+        }
+        //this adds vehicle type to carlot class list
+        public void addVehicle (Vehicle myVehicle) {
+            numofCars.Add (myVehicle);
+
+        }
+        public void lotInventory () 
+        {
+            Console.WriteLine ("There are {0} on the CarLot.", numofCars.Count);
+            
+            //looping through all cars
+            foreach (Vehicle item in numofCars) {
+                item.printDescription ();
+                Console.WriteLine ("");
+            }
+        }
+
+    }
+
+    //abstract vehicle class, cannot be instantiated
+    public abstract class Vehicle {
+        public string licenseNumber;
+        public string vehicleMake;
+        public string vehicleModel;
+        public double vehiclePrice;
+        public string vehicleDescription;
+
+        //this method printsdetails about all vehicles
+        virtual public void printDescription () {
+            Console.WriteLine ("Vehicle Description: {0}", this.vehicleDescription);
+            Console.WriteLine ("License number : {0}", this.licenseNumber);
+            Console.WriteLine ("Make : {0}", this.vehicleMake);
+            Console.WriteLine ("Model : {0}", this.vehicleModel);
+            Console.WriteLine ("The price of of the vehicle is ${0}", this.vehiclePrice);
+        }
+
+    }
+
+    // overriding from the base object class
+    class Car : Vehicle
+    //listing properties of a car
+    {
+        
+        int numberofDoors = 2;
+        string vehicleType = "";
+
+          //constructor for car
+        public Car (string description, string license, string make, string model, string Type, int numberofDoors, double Price)
+
+        {
+            this.licenseNumber = license;
+            this.vehicleMake = make;
+            this.vehicleModel = model;
+            this.vehiclePrice = Price;
+            this.vehicleType = description;
+        }
+        public override void printDescription () {
+            
+
+            Console.WriteLine ("Vehicle Description: {0}", this.vehicleDescription);
```

Comment:
You can save some duplicate code by calling `base.printDescription()` instead of duplicating the printing of license, make, model, price, and type.